python/configuration_manager/directories_manager.py

```diff
@@ -4,9 +4,11 @@
 #  Version 2.0."
 
 import os
-from utils.util import safe_makedir
+# TODO: relative import instead of absolute import
+# from utils.util import safe_makedir
+from python.configuration_manager.utils import util
```

Comment:
Even though the packages is called **utils**, imported module could be called **utils** as well.